tests/shared/common.go

```diff
@@ -144,6 +144,84 @@ func MakeModule(configVersion string) csmv1.Module {
 	return moduleObj
 }
 
+func MakeSecretPowerFlexWithZone(name, ns, _ string) *corev1.Secret {
+	dataWithZone := `
+- username: "admin"
+  password: "password"
+  systemID: "2b11bb111111bb1b"
+  endpoint: "https://127.0.0.2"
+  skipCertificateValidation: true
+  mdm: "10.0.0.3,10.0.0.4"
+  zone:
+    name: "US-EAST"
+    labelKey: "zone.csi-vxflexos.dellemc.com"
+`
+	secret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: ns,
+		},
+		Data: map[string][]byte{
+			"config": []byte(dataWithZone),
+		},
+	}
+	return secret
+}
+
+// MakeSecret  returns a driver pre-req secret array-config
+func MakeSecretPowerFlex(name, ns, _ string) *corev1.Secret {
+	dataWithoutZone := `
+- username: "admin"
+  password: "password"
+  systemID: "2b11bb111111bb1b"
+  endpoint: "https://127.0.0.2"
+  skipCertificateValidation: true
+  mdm: "10.0.0.3,10.0.0.4"
+`
+
+	secret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: ns,
+		},
+		Data: map[string][]byte{
+			"config": []byte(dataWithoutZone),
+		},
+	}
+	return secret
+}
+
+func MakeSecretPowerFlexMultiZoneInvalid(name, ns, _ string) *corev1.Secret {
```

Comment:
Please add function description above the method definition

Reply:
done.